bootcamp/tutorials/integration/milvus_and_DSPy.ipynb

```diff
@@ -0,0 +1,873 @@
+{
```

Comment:
<p>Can you zoom out the picture a bit? It's taking too much space.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
solved